controllers/companyController.js

```diff
@@ -156,129 +156,118 @@ function setupPassword(){
-                            username:username,
-                            password:password
-                        })
-                        console.log(NewUser);
-                        NewUser.save(function(err){
-                          //set up email content
-                            const mailOptions = {
-                                from: 'itpsychiatrist.policymanager@gmail.com', // sender address
-                                to: NewCompany.company_email, // list of receivers
-                                subject: 'Your IT Policy Manager Login Credentials', // Subject line
-                                html: '<h2>Welcome to IT Policy Manager!</h2>' + '<p> Thank you for registering. <br>' +
-                                'Below is your login details, use these credential to acces your IT Policy Manager Account.<br><br>' +
-                                'User Name:' + NewUser.username + ' <br> Temporary Password: ' + NewUser.password + ' </p>' +
-                                '<p>Please click on the link to sign-in. </p> <a href="http://localhost:3000/signin-page">IT Policy Manager Login</a>  '
-                            };
-                            transporter.sendMail(mailOptions, function (err, info) {
-                                if(err)
-                                console.log(err)
-                                else
-                                console.log(info);
-                            });
-
-
-                        })
-                        res.json({
-                            message: "Registration Successful!",
-                            value:true,
-                            id: companyId
-                        })
-                        console.log("res: "+res.id);
-                    }
+                console.log(NewUser);
+                NewUser.save(function (err) {
+                    postCreateNewUser(err)
                 })
-            }else{
                 res.json({
-                    message: "Company already existed! Login Instead",
-                    value:false
+                    message: "Registration Successful!",
+                    value: true,
+                    id: companyId
                 })
-            }
-        });
-    };
-    
+                console.log("res: " + res.id);
+            })
+        }
+    });
 };
 
-exports.companyDelete=(req,res)=>{
-   let data=req.body;
-   Company.findByIdAndRemove({
-       _id:data.companyId
-   },function(error,response){
-       if(!error){
-           User.findOneAndRemove({company:data.companyId},function(error2,response2){
-               if(!error2){
-                res.json({
-                    status:"success"
-                })
-               }
-               else{
-                   console.log(error2)
-               }
-           })
-           
-       }
-       else{
-           console.log(error)
-       }
-   })   
+function postCreateNewUser(err) {
```

Comment:
Rename the function according to what it does.
Maybe 'mailAboutUserCreation' or 'notifyAboutUserCreation' or 'writeUserBeenCreatedEmail'
Or something better.